pom.xml

```diff
@@ -294,6 +294,27 @@
         <spotless.check.skip>true</spotless.check.skip>
       </properties>
     </profile>
+    <profile>
+      <id>profiler</id>
+      <activation>
+        <property> <!-- with this the profiler extension is activated too -->
+          <name>profiler</name>
+          <value>true</value>
+        </property>
+      </activation>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-surefire-plugin</artifactId>
+            <version>${surefire.version}</version>
+            <configuration>
+              <reportFormat>plain</reportFormat>
+            </configuration>
+          </plugin>
+        </plugins>
+      </build>
+    </profile>
```

Comment:
I think it may be useful to run this on the CI by default. WDYT?

Reply:
this will report the durations of individual tests, but also for every maven plugin and maven project. Surefire will write to console, which makes the output unreadable again, but AFAIK this info is already logged to file as well. 
If nobody cares I would keep if off, and activate it once you need to analyse build times again.